app/src/main/res/layout/screen_access_denied.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<org.simple.clinic.deniedaccess.AccessDeniedScreen android:id="@+id/applock_root"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:background="@color/window_background_auth"
+  tools:ignore="Overdraw">
+
+  <LinearLayout
```

Comment:
We've been using `ConstraintLayout` when building new screens since it makes changing screens in the future easier. Should we do the same here as well?

Reply:
As discussed, we will keep this layout as is for now since we don't expect this layout to change a lot.